src/reducers/testReducer.js

```diff
@@ -0,0 +1,31 @@
+import { TEST } from "../actions/types";
+
+const initialState = {
+  isState: false,
+  message: "Remove me"
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case `${TEST}_PENDING`:
```

Comment:
Please refactor this into a function, because we will be doing this a lot moving forward.
create a file e.g `actionUtil.js` in this file define a function to append the `_PENGING` `_FAILED` `_FULFILLED` to the action type. e.g 
``` js

export const PENDING = type => `${type}_PENDING` 

```